wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Comments: add detail list view

Fixes #11160 

On the Insights Comments card, selecting `View more` will now show a full list of Comments.
- Currently, there is no API call to get the full list of Comments for either tab. So for now, this displays the data fetched from the "top" Comments API call used for the Insights Comments card. (cc @jklausa  issue #11182)
- When `View more` is selected, the filter tab (and thus the data) will be the tab selected on the overview card.
- Pull to refresh updates the data.
- If one tab has data and the other does not, the empty tab will display `No data yet`.

NOTE: The UI for the details view still needs to be updated. That will be done with https://github.com/wordpress-mobile/WordPress-iOS/issues/11165.

To test:

---
- On a site with more than 6 comments, go to Insights > Comments > View more.
- Verify:
  - The tab titles match the overview card.
  - The selected tab match the overview card.

![comments_list](https://user-images.githubusercontent.com/1816888/53756420-5420a780-3e76-11e9-9511-e5a5ee4aaabc.png)

---
- On the Posts and Pages tab, selecting a row opens a web view for that post/page.

---
